src/storage/table.cc

```diff
@@ -338,28 +338,24 @@ void DBTable::InsertRecord(std::vector<std::string_view> values,
 }
 
 bool DBTable::GetMetadataOk() {
-  if (!metadata_enabled) {
-    return true;
-  } else {
-    for (int i = 0; i < get_num_blocks(); i++) {
-      bool block_compatible = get_block(i)->IsMetadataCompatible();
-      if (!block_compatible) {
-        // ignore blocks that are not metadata compatible
-      } else {
+  if (metadata_enabled) {
+    for (size_t i_block = 0; i_block < get_num_blocks(); i_block++) {
```

Comment:
Another clean way of renaming the iterator could be something like this,

**block_idx** or **col_idx**